packages/elements/src/components/API/__tests__/utils.test.ts

```diff
@@ -326,3 +326,117 @@ describe('computeTagGroups', () => {
     expect(serviceNode ? computeTagGroups(serviceNode) : null).toEqual({ groups: [], ungrouped: [] });
   });
 });
+
+describe('computeAPITree', () => {
+  it('generates API ToC tree', () => {
+    const apiDocument: OpenAPIObject = {
+      openapi: '3.0.0',
+      info: {
+        title: 'some api',
+        version: '1.0.0',
+        description: 'some description',
+      },
+      paths: {
+        '/something': {
+          get: {
+            responses: {
+              200: {
+                schema: { $ref: '#/definitions/schemas/ImportantSchema' },
+              },
+            },
+          },
+        },
+      },
+      components: {
+        schemas: {
+          ImportantSchema: {
+            type: 'object',
+            properties: {
+              a: { type: 'string' },
+            },
+          },
+        },
+      },
+    };
+
+    expect(computeAPITree(transformOasToServiceNode(apiDocument)!)).toEqual([
+      {
+        id: '/',
+        meta: '',
+        slug: '/',
+        title: 'Overview',
+        type: 'overview',
+      },
+      {
+        title: 'Endpoints',
+      },
+      {
+        id: '/paths/something/get',
+        meta: 'get',
+        slug: '/paths/something/get',
+        title: '/something',
+        type: 'http_operation',
+      },
+      { title: 'Schemas' },
+      {
+        id: '/schemas/ImportantSchema',
+        slug: '/schemas/ImportantSchema',
+        title: 'ImportantSchema',
+        type: 'model',
+        meta: '',
+      },
+    ]);
+  });
+
+  it('hides schemas from ToC', () => {
```

Comment:
*when instructed* or something. It doesn't hide them by default.

Reply:
Yup, done 👍